src/components/rating/Rating.spec.jsx

```diff
@@ -156,44 +158,111 @@ describe('rating', () => {
     });
   });
 
-  test('counter', () => {
-    const rating = mount(
-      <Rating counter={30} />
-    );
+  describe('counter text', () => {
+    it('exist', () => {
+      const rating = shallow(
+        <Rating />
+      );
+      const rateCounter = rating.find(RateCounter);
 
-    expect(rating.find('.sg-rate-box__counter')).toHaveLength(1);
-  });
+      expect(rateCounter).toHaveLength(1);
+    });
 
-  test('no counter', () => {
-    const rating = mount(
-      <Rating />
-    );
+    it('displays active text when active and haven\'t been rated', () => {
+      const rating = shallow(
+        <Rating active />
+      );
+      const rateCounter = rating.find(RateCounter);
+
+      expect(rateCounter.props().showActiveText).toBeTruthy();
+    });
+
+    it('displays counter text when no active and haven\'t been rated', () => {
+      const rating = shallow(
+        <Rating />
+      );
+      const rateCounter = rating.find(RateCounter);
+
+      expect(rateCounter.props().showActiveText).toBeFalsy();
+    });
+
+    it('displays counter text when have been rated and no active', () => {
+      const rating = shallow(
+        <Rating rate={3} />
+      );
+      const rateCounter = rating.find(RateCounter);
 
-    expect(rating.find('.sg-rate-box__counter')).toHaveLength(0);
+      expect(rateCounter.props().showActiveText).toBeFalsy();
+    });
+
+    it('displays counter text when have been rated and active', () => {
+      const rating = shallow(
+        <Rating rate={3} active />
+      );
+      const rateCounter = rating.find(RateCounter);
+
+      expect(rateCounter.props().showActiveText).toBeFalsy();
+    });
+
+    // check if let is needed, looks like a bug on enzyme side
+    it('displays active text when active and mouse over stars', () => {
+      const rating = mount(
+        <Rating rate={3} active />
+      );
+      const stars = rating.find('.sg-rate-box__stars');
+      let rateCounter = rating.find(RateCounter);
+
+      expect(rateCounter.props().showActiveText).toBeFalsy();
+
+      stars.simulate('mouseEnter');
+      rateCounter = rating.find(RateCounter);
+      expect(rateCounter.props().showActiveText).toBeTruthy();
+
+      stars.simulate('mouseLeave');
+      rateCounter = rating.find(RateCounter);
+      expect(rateCounter.props().showActiveText).toBeFalsy();
+    });
+
+    // should be near same as above, so if we let delete above we need delete it here
+    it('no displays active text when no active and mouse over stars', () => {
```

Comment:
`doesn't display`, we still use english grammar :)

Reply:
all tests description updated